source/client/factories_impl.cc

```diff
@@ -31,18 +31,26 @@ BenchmarkClientPtr BenchmarkClientFactoryImpl::create(
     Envoy::Api::Api& api, Envoy::Event::Dispatcher& dispatcher, Envoy::Stats::Scope& scope,
     Envoy::Upstream::ClusterManagerPtr& cluster_manager,
     Envoy::Tracing::HttpTracerSharedPtr& http_tracer, absl::string_view cluster_name,
-    RequestSource& request_generator) const {
+    int sink_stat_prefix, RequestSource& request_generator) const {
```

Comment:
it's possible this is also true elsewhere in this PR, but is it desirable for us to only allow an int as the sink_stat_prefix?  In the future, isn't it possible that we'd want to use a prefix that isn't just the worker number?

If not, then this really isn't just a prefix.  It has the specific meaning of being a worker number.

Reply:
Discussed offline. Decide to change sink_stat_prefix to worker_id to make it explicitly that this will be used to populate worker_id field in `SinkableHdrStatistic`
https://github.com/envoyproxy/nighthawk/blob/d8f4d310040d1f99eb78d0ede8d9f616308eb8bf/source/client/factories_impl.cc#L45-L50

\cc @oschaaf